core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -289,6 +295,66 @@ static ClientRequestContextBuilder builder(RpcRequest request, URI uri) {
     @Override
     RpcRequest rpcRequest();
 
+    /**
+     * Pushes the specified context to the thread-local stack. To pop the context from the stack, call
+     * {@link SafeCloseable#close()}, which can be done using a {@code try-with-resources} block:
+     * <pre>{@code
+     * try (PushHandle ignored = ctx.push(true)) {
```

Comment:
`SafeCloseable` (seems to be all I talk about lately :) )

Reply:
Oops didn't notice it. Thanks!